### ISSUE TITLE
[NN] write neural network

### ISSUE THREAD

**Issue by zekmaro** [zekmaro/multilayer-perceptron#2]
1. build layers and neurons
2. calculate activations and biases
3. make loss
4. backpropagate
